liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/StartDebugMojoSupport.java

```diff
@@ -289,10 +318,13 @@ protected void runLibertyMojoDeploy(boolean forceLooseApp) throws MojoExecutionE
         if(forceLooseApp) {
             Xpp3Dom looseApp = config.getChild("looseApplication");
             if (looseApp != null && "false".equals(looseApp.getValue())) {
-                log.warn("Overriding liberty plugin pararmeter, \"looseApplication\" to \"true\" and deploying application in looseApplication format");
+                log.warn("Overriding liberty plugin parameter, \"looseApplication\" to \"true\" and deploying application in looseApplication format");
                 looseApp.setValue("true");
             }
         }
+        if (LooseWarApplication.isExploded(project)) {
+			runMojo("org.apache.maven.plugins", "maven-resources-plugin", "resources");
+        }
```

Comment:
Seems like we could be in this method for both a `war` and an `ear` application, in which case is it ok to call `isExploded` on `LooseWarApplication`?

Reply:
Done